<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMediaWindow</class>
 <widget class="QWidget" name="CMediaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Information</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="externalUrls">
     <property name="text">
      <string>External Urls:</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="3">
    <widget class="CMediaUserDataWidget" name="lhsUserMediaData">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>LHS User Data:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="applyToRight">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="EmbySync.qrc">
       <normaloff>:/resources/arrowright.png</normaloff>:/resources/arrowright.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="2" rowspan="3">
    <widget class="CMediaUserDataWidget" name="rhsUserMediaData">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>RHS User Data:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QToolButton" name="applyToLeft">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="EmbySync.qrc">
       <normaloff>:/resources/arrowleft.png</normaloff>:/resources/arrowleft.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="currMediaName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Media Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="currMediaType">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="process">
       <property name="text">
        <string>&amp;Process</string>
       </property>
       <property name="icon">
        <iconset resource="EmbySync.qrc">
         <normaloff>:/resources/upload.png</normaloff>:/resources/upload.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMediaUserDataWidget</class>
   <extends>QGroupBox</extends>
   <header>MediaUserDataWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="EmbySync.qrc"/>
 </resources>
 <connections/>
</ui>
